Gruntfile.js

```diff
@@ -8,6 +7,15 @@ const path = require('path');
 
 const kAllSuites = ['webgpu', 'stress', 'manual', 'unittests', 'demo'];
 
+const kFilesForEslint = [
+  // TS
+  'src/**/*.ts',
+  // JS
+  '*.js',
+  '.*.js',
+  'tools/**/*.js',
```

Comment:
`setup-ts-in-node.js` is not linted because for some reason it insisted on parsing it as a TS file even though it is not.

I also wanted to lint JSON files but this would require a plugin so I didn't bother.